docs/references/vql.yaml

```diff
@@ -789,6 +789,15 @@
     description: Rc4 key (1-256bytes).
     required: true
   category: plugin
+- name: entropy
+  description: Calculates shannon scale entropy of a string.
+  type: Function
+  args:
+  - name: String
```

Comment:
You actually don't need to write this by hand, just run make reference to have it automatically done.

In any case the arg name is string with a lower case.

Reply:
Good to know!
I will use make reference to update the file.